lib/groonga/client/response/error.rb

```diff
@@ -66,6 +67,21 @@ def line
           end
         end
       end
+
+      class InvalidResponse < Client::Error
+        attr_reader :command
+        attr_reader :raw_response
+        def initialize(command, raw_response, error_message)
+          @command = command
+          @raw_response = raw_response
+          message = +"invalid response: "
+          message << "#{command.command_name}: "
+          message << "#{error_message}: "
+          message << command.to_command_format
+          message << ": <#{raw_response}>"
```

Comment:
実際のメッセージを見ると読みにくいからちょっと印を入れようか。

```suggestion
          message << "<#{command.to_command_format}>: "
          message << "<#{raw_response}>"
```